frontend/src/app/choreography/choreography.component.ts

```diff
@@ -498,4 +500,22 @@ export class ChoreographyComponent {
     }
     this.activeChoreographyItem = item;
   }
+
+  saveToDatabase(): void {
+    const dialogRef = this.dialog.open(SaveChoreographyDialogComponent, {});
+    dialogRef
+      .afterClosed()
+      .pipe(filter((result) => !!result))
+      .subscribe(() => {
+        console.log('hello');
+        this.http
+          .post<Choreography>(`/api/choreographies`, {
+            ...this.choreography,
+            id: 4, // Koreografia -tyypissä on id, mutta olisi parempi että tietokanta loisi sen. Id 4 on testausta varten.
+            frames: JSON.stringify(this.choreography.frames),
```

Comment:
**suggestion:** better to do this at the backend. Doesn't seem sensible for the frontend to send a string value as part of the payload.

Reply:
Alright, I'll move the manipulation of the payload to the backend.